packages/ui/src/working-groups/components/ApplicantsList.tsx

```diff
@@ -17,6 +17,8 @@ export interface WorkersListProps {
 }
 
 export const ApplicantsList = ({ hired, allApplicants, myApplication, hiringComplete, leadId }: WorkersListProps) => {
+  allApplicants = allApplicants?.filter((application) => application.status != 'ApplicationStatusWithdrawn')
```

Comment:
```suggestion
  allApplicants = allApplicants?.filter((application) => application.status !== 'ApplicationStatusWithdrawn')
```

Reply:
> Thanks for PR! Since @theo is overloaded I will try to cover this PR for him.
> 
> I think it would be good to cover filtering out withdrawn applications in parent component of `ApplicantsList` under `useOpening`. The reason behind is that `myApplication` is currently resolved with `find` function on not filtered applications. In the example with isonar it may cause situation when it will find his withdrawn application first and display wrong application.
> 
> Alternatively, instead of bringing filtering to parent, we can just add extra condition inside `find` for `myApplication`.

Thanks for the review!
yeah well notice, I moved the filtering to the parent component, as you suggested, to avoid that problem